src/cascadia/WindowsTerminal/TrayIcon.cpp

```diff
@@ -23,6 +23,37 @@ TrayIcon::~TrayIcon()
     RemoveIconFromTray();
 }
 
+void TrayIcon::CreateWindowProcess()

```

Comment:
```suggestion
void TrayIcon::CreateWindow()
```

This is sufficient -- it's not a process, after all! :smile: